controlplane/pkg/apis/remote/connection/helpers.go

```diff
@@ -54,7 +54,7 @@ func (c *Connection) SetContext(newContext *connectioncontext.ConnectionContext)
 	c.Context = newContext
 }
 
-// IsComplete - Have I been told enough to actually give you what you asked for
+// IsValid - Have I been told enough to actually give you what you asked for
```

Comment:
In case of rename, comment should be also changed.

Reply:
oops! Thanks, fixed.